_data/settings/components/accordion.yml

```
@@ -1,5 +1,10 @@
 name: Accordion
 contents:
+  - name: Background color
+    description: Content background color of the accordion.
```

A possible tiny tweak:
```suggestion
    description: Background color of the accordion content.
```